uint64 timeStamp
uint16 nSignals
uint16 hopSize
uint16 fs
float32[] data
